Source/Core/InputCommon/ControllerEmu/ControllerEmu.h

```diff
@@ -103,13 +103,15 @@ struct ThreePointCalibration
 template <typename T, size_t Bits>
 struct RawValue
 {
-  RawValue() = default;
-  explicit RawValue(const T& value_) : value{value_} {}
+  constexpr RawValue() = default;
+  constexpr explicit RawValue(const T& value_) : value{value_} {}
 
   static constexpr size_t BITS_OF_PRECISION = Bits;
 
   T value;
 
+  constexpr bool operator==(const RawValue& other) const { return value == other.value; }
```

Comment:
(Could even do `constexpr bool operator==(const RawValue& other) const = default;` since RawValue is not an union, but I suppose it's just as easy to write the comparison explicitly here)

Reply:
Sure.